### PR TITLE
update docs following production cluster turn up

### PR DESCRIPTION
**Pull request by fredericfran-gds**
Main changes:
1. move the terraform section from `create-a-new-environment.md` and consolidate in `applying-terraform.md`
2. update docs about secrets to include all 3 types of secrets and information about format of the platform secrets
3. link in `applying-terraform.md` that there needs to bootstrap signon/signon-resources

### CODE REVIEW

File: terraform/docs/applying-terraform.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ The EKS cluster is deployed via Terraform in two stages. See [adr-3] for backgro
 - `cluster-infrastructure` is concerned only with setting-up the EKS cluster and associated AWS resources (such as the worker groups and auto-scaling groups).
 - `cluster-services` is concerned only with setting up the Kubernetes resources and configuration for base services, including the `aws-auth` ConfigMap, ingress controller, etc.
 
+## Prerequisites
+
+1. Some of the deployments below require external [tfvars
+files](../terraform/deployments/variables). To create a new environment, you
+will need to copy the variables directory for an existing environment (e.g.
+[integration](../deployments/variables/integration)) and modify as
```
Comment (nsabri1) [inline terraform/docs/applying-terraform.md:13]: i wonder if we should mention to copy name it to the name of the new environment, such as TEST etc...?

**Review by nsabri1 — APPROVED** [review]
Couple comments - But LGTM, thanks

File: terraform/docs/applying-terraform.md (hunk 1 above)
Comment (fredericfran-gds) [reply terraform/docs/applying-terraform.md:13]: I think this is what is saying, copy the directory

File: docs/prerequisite-secrets.md [hunk 2]
```diff
@@ -19,53 +33,114 @@ The format of secret is given below to aid creation from scratch:
...
    {
-     clientID: <secret_1>,
-     clientSecret: <secret_2>
+     "clientID": "<secret_1>",
+     "clientSecret": "<secret_2>"
    }
    ```
 
-## Manually generated secrets
+2. `govuk/logit-host`: used by FileBeat in k8s to access the Logit stack.
```
[the remark this answers is not in the document]
Comment (fredericfran-gds) [reply docs/prerequisite-secrets.md:52]: I can do that

**Pull request merged**